x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -45,19 +45,29 @@ type MisconfigurationSortFieldType =
   | 'resource'
   | 'rule';
 
+interface MisconfigurationDetailsDistributionBarProps {
+  key: string;
+  count: number;
+  color: string;
+  filter: () => void;
+  isCurrentFilter: boolean;
+  reset: (event: React.MouseEvent<SVGElement, MouseEvent>) => void;
+}
+
 const getFindingsStats = (
   passedFindingsStats: number,
   failedFindingsStats: number,
   filterFunction: (filter: string) => void,
   currentFilter: string
 ) => {
+  const misconfigurationStats: MisconfigurationDetailsDistributionBarProps[] = [];
   if (passedFindingsStats === 0 && failedFindingsStats === 0) return [];
-  return [
-    {
+  if (passedFindingsStats > 0) {
+    misconfigurationStats.push({
       key: i18n.translate(
         'xpack.securitySolution.flyout.right.insights.misconfigurations.passedFindingsText',
         {
-          defaultMessage: 'Passed findings',
+          defaultMessage: passedFindingsStats > 1 ? 'Passed findings' : 'Passed finding',
```

Comment:
I'm afraid that won't work with translations, pluralization with i18n needs to follow [this](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/GUIDELINE.md#pluralization) guideline:


```suggestion
          defaultMessage: 'Passed {passedFindingsStats, plural, one {# finding} other {# findings}}',
          values: { passedFindingsStats },
```

Reply:
ooo I tried to do this originally, but I must have gotten the syntax wrong